.circleci/config.yml

```
@@ -293,6 +298,70 @@ jobs:
           working_directory: ~/pix/mon-pix
           command: npm ci
 
+      - run:
+          name: Start Pix API
+          working_directory: ~/pix/api
+          environment:
+            DATABASE_URL: 'postgres://circleci@localhost:5432/circleci'
+            REDIS_URL: 'redis://localhost:6379'
+          background: true
+          command: npm start
+
+      - run:
+          name: Start Pix App
+          working_directory: ~/pix/mon-pix
+          environment:
+            JOBS: 1
+          background: true
+          command: npm start
+
+      - run:
+          name: Run tests
+          environment:
+            DATABASE_URL: 'postgres://circleci@localhost:5432/circleci'
+          command: npm run test:ci:app
+      - store_test_results:
+          path: /home/circleci/test-results
+
+  e2e_test_orga:
+    docker:
+      - image: cimg/node:16.19.0-browsers
+      - image: postgres:14.6-alpine
+        environment:
+          POSTGRES_USER: circleci
+          POSTGRES_HOST_AUTH_METHOD: trust
+      - image: redis:6.2.7-alpine
```

suggestion: on pourrait definir un executor (j'ai pas testé mais je peux :D)
```suggestion
executors:
  docker-front-executor:
    docker:
      - image: cimg/node:16.19.0-browsers
      - image: postgres:14.6-alpine
        environment:
          POSTGRES_USER: circleci
          POSTGRES_HOST_AUTH_METHOD: trust
      - image: redis:6.2.7-alpine
  
 e2e_test_orga:
    executor: docker-front-executor
```

---

Ça me semble être une bonne idée, plutôt dans une autre PR.